recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
 {% set name = "mamba" %}
-{% set libmamba_version = "0.19.1" %}
-{% set libmambapy_version = "0.19.1" %}
-{% set mamba_version = "0.19.1" %}
-{% set release = "2021.12.08" %}
+{% set libmamba_version = "0.22.1" %}
```

Comment:
Is there any chance that all these 3 versions can be ever different for a particular mamba version? If not, we should define a single version variable.